service/openapi.yaml

```diff
@@ -279,3 +314,40 @@ components:
     errorDetails:
       type: string
       description: An error description for logging or other needs
+    metadataResponse:
+      description: OFREP metadata response
+      properties:
+        name:
+          type: string
+          description: name of the flag management system
+          examples:
+            - flagd
+            - go-feature-flag
+        features:
+          type: object
+          description: List of the features supported by the flag management system.
+          properties:
+            cacheInvalidation:
+              $ref: '#/components/schemas/featureCacheInvalidation'
+    featureCacheInvalidation:
+      type: object
+      description: Configuration for the cache cacheInvalidation
+      properties:
+        polling:
+          $ref: '#/components/schemas/featureCacheInvalidationPolling'
+    featureCacheInvalidationPolling:
+      type: object
+      description: Configuration of the polling for the featureCacheInvalidation
+      properties:
+        enabled:
+          type: boolean
+          description: set to true if the remote flag management system is supporting polling
+          default: false
+        minPollingInterval:
+          type: number
+          description: minimum polling interval (in millisecond) supported by the flag management system. The provider should ensure not to set any polling value under this minimum.
+          examples:
+            - 60000
+          default: 60000
```

Comment:
I am not sure about having a default value here.
For a protocol implemented by different vendors and projects, I think having default for timing that are not technically constrained is not too helpful.
Is there any specific reason for this value @thomaspoignant?

Reply:
It was to be explicit on what the provider should implement as a default value.
It is also based on @beeme1mr comment https://github.com/open-feature/protocol/pull/9#discussion_r1556377284.

Reply:
I see, did not see that this was the motivation for the default.
As I understood @beeme1mr he would not expect a default but rather a behavior if it is omitted.
I would also not add a default value for this specific parameter, as I think it just depends in this case.

Reply:
I don't have a strong feeling to keep it or not.
We can remove it if you feel that it is needed.

For the omitted behaviour, I am not sure if OpenAPI has a way to represent that?

Reply:
Mh I think we can only describe the omitted behavior.
I am more against having a default value for the interval as it is so dependent from the concrete system and use case, and I don't see a way of finding a "good" default the one could argue about.

Reply:
I'm in favor of removing the defaults but I think we'll need a separate document describing the expected behavior of the providers.

Reply:
For example, we need to define what happens in the provider if a user defines a polling interval faster than the flag management tools allow.

Reply:
> I'm in favor of removing the defaults but I think we'll need a separate document describing the expected behavior of the providers.

This may be something we add in the ADRs?
Would it work for you?

Reply:
It's more of an OFREP provider spec than an architecture decision. I was thinking about something like this 
https://flagd.dev/reference/specifications/in-process-providers/

Reply:
> It's more of an OFREP provider spec than an architecture decision.

Yes I would say so too.
I think it should be a provider spec, as OFREP and the OpenAPI Docs are somewhat disconnected from that.

Reply:
Yes, that was the initial plan we had with @Kavindu-Dodan.
I'll try to have a 1st draft tonight.

Reply:
Awesome, thanks!

Reply:
+1 for defining default values in the spec. 

Besides, we can make the metadata endpoint optional as the spec will define the defaults.